Assets/Scripts/Health/AnimalHealth.cs

```diff
@@ -31,4 +39,19 @@ void Update()
             Destroy(gameObject);
         }
     }
+    void PlayHurtSound()
+    {
+        if (audioSourceHurt != null && !audioSourceHurt.isPlaying)
+        {
+            int n = Random.Range(1, koalaHurtSound.Length);
+            audioSourceHurt.clip = koalaHurtSound[n];
+            audioSourceHurt.Play();
+            koalaHurtSound[n] = koalaHurtSound[0];
```

Comment:
why do we need these two lines?

             koalaHurtSound[n] = koalaHurtSound[0];
             koalaHurtSound[0] = audioSourceHurt.clip;

Reply:
We don't really "need" them but they are there to prevent playing the same sound multiple times in a row. It can get annoying if the randomization picks the same sound 3+ times in a row.